Cargo.toml

```diff
@@ -18,10 +18,25 @@ exclude = ["/.*"]
 event-listener = { version = "5.0.0", default-features = false }
 event-listener-strategy = { version = "0.5.0", default-features = false }
 pin-project-lite = "0.2.11"
+portable-atomic-util = { version = "0.1.4", default-features = false, optional = true, features = ["alloc"] }
+
+[dependencies.portable_atomic_crate]
+package = "portable-atomic"
+version = "1.2.0"
+default-features = false
+optional = true
+
+[target.'cfg(loom)'.dependencies]
+loom = { version = "0.7", optional = true }
 
 [features]
 default = ["std"]
+portable-atomic = ["portable-atomic-util", "portable_atomic_crate"]
```

Comment:
I saw the changes to lib.rs, but this doesn't really seem to be in use?

Reply:
Whoops, I was going to add a `portable-atomic` impl but forgot to. Will fix this.